rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init

```diff
@@ -390,6 +390,10 @@ start_slapd() {
       MEMSIZE=$(sed -n '/^MemTotal/s/[^0-9]//gp' /proc/meminfo)
       # Round down using integer division (800 bytes is the size of an entry)
       CACHESIZE=$((${MEMSIZE} * 1024 / 800 / 10 / 10000 * 10000))
+      # Maximum cache size of 40M entries. Above about 40G of RAM (about 53.6M entries) OpenLDAP dumps core, so we force an upper limit.
```

Comment:
40M entries is about 30G of RAM to be coherent with the 800 bytes estimation above.